crates/proof-of-sql/src/base/scalar/mont_scalar.rs

```diff
@@ -649,3 +649,13 @@ where
         BigInt::from_bytes_le(sign, bits)
     }
 }
+
+impl<T> From<&Vec<u8>> for MontScalar<T>
+where
+    T: MontConfig<4>,
+    MontScalar<T>: Scalar,
+{
+    fn from(value: &Vec<u8>) -> Self {
+        MontScalar::from_le_bytes_mod_order(value.as_slice())
+    }
+}
```

Comment:
We should remove this. There are multiple reasonable embeddings from bytes to a scalar. We shouldn't pick one of them as the "default".